framework/src/media/MediaWorker.cpp

```diff
@@ -40,13 +40,13 @@ void MediaWorker::startWorker()
 		pthread_attr_t attr;
 		pthread_attr_init(&attr);
 		pthread_attr_setstacksize(&attr, mStacksize);
-		mIsRunning = true;
```

Comment:
The mIsRunning variable is used in the thread loop for checking the thread is alive. This line should be remain.